dll/ime/msctfime/lang/pt-PT.rc

```diff
@@ -0,0 +1,24 @@
+/*
+ * PROJECT:     ReactOS msctfime.ime
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
+ */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_OK               "OK"
+    IDS_CANCEL           "Cancelar"
+    IDS_ABORT            "&Abortar"
+    IDS_RETRY            "&Retroceder"
+    IDS_IGNORE           "&Ignorar"
+    IDS_YES              "&Sim"
+    IDS_NO               "&Não"
+
+    IDS_ENTER            "Enter"
+    IDS_FINALIZE_STRING  "Finalizar cadeia"
+    IDS_CONVERSION       "Conversão"
+END
+
```

Comment:
Excess line at the end.

Reply:
At https://reactos.org/wiki/Translation_Introduction
it says: Leave an end of file line (a blank line) at the end of your rc file for ease of building.

Reply:
Yes, but you have two lines, one of which is excessive.

Reply:
You're right. My apologies.